dist/angular-promise-buttons.js

```diff
@@ -2,7 +2,7 @@ angular.module('angularPromiseButtons', [
 ]);
 
```

Comment:
Well, I was doing grunt serve to test. It's not like I was editing this file by hand. :)